src/main/java/uk/gov/hmcts/reform/unspec/service/tasks/handler/CaseEventTaskHandler.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.unspec.service.tasks.handler;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.camunda.bpm.client.task.ExternalTask;
+import org.camunda.bpm.client.task.ExternalTaskHandler;
+import org.camunda.bpm.client.task.ExternalTaskService;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.unspec.callback.CaseEvent;
+import uk.gov.hmcts.reform.unspec.model.BusinessProcess;
+import uk.gov.hmcts.reform.unspec.service.CoreCaseDataService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.unspec.helpers.ExponentialRetryTimeoutHelper.calculateExponentialRetryTimeout;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class CaseEventTaskHandler implements ExternalTaskHandler {
+
+    private final CoreCaseDataService coreCaseDataService;
+
+    @Override
+    public void execute(ExternalTask externalTask, ExternalTaskService externalTaskService) {
+        final String taskName = externalTask.getTopicName();
+        log.info("Job {} started", taskName);
```

Comment:
can you think of a way we can make this logging generic? It's going to be used in a similar way for every camunda interaction, scheduler etc 🤔

Reply:
done